### DISCUSSION TITLE
How to use Binding Variables ?

### DISCUSSION THREAD

**Discussion by MuelPas21** [vtulluru/node-red-contrib-oracledb-mod#36] [Q&A]
I Want to use Binding Variables bsp.: Select * From Station Where CELL_NAME = :Name

How do i Pass the PArameter for the Bindung :Name into the Node?

Tanks in Advance 
Pascal

**Comment by tlraghavendra** [comment]
Hi,

I hope it is not so late,

Use a function node at first

`
var payload = [];
payload = ["YOUR_Name"];
var qry = ``Select * From Station Where CELL_NAME = :1``;
msg.query = qry;
msg.payload = payload;
return msg;
`
and next add Oracledb node will do.

![image](https://github.com/vtulluru/node-red-contrib-oracledb-mod/assets/5299237/dc8c614d-f6ab-4dcf-9202-09f9e70f2b52)
